### DISCUSSION TITLE
Failure Starting Docker Container

### DISCUSSION THREAD

**Discussion by travisanderson65** [dani-garcia/vaultwarden#2216] [Q&A]
I am getting the following error when I run the the command "docker run -d --name vaultwarden -v /vw-data/:/data/ -p 80:80 vaultwarden/server:latest"

Error: 
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "proc" to rootfs at "/proc" caused: mount through procfd: permission denied: unknown.
docker container ls --all
CONTAINER ID   IMAGE                       COMMAND                  CREATED              STATUS    PORTS                                         NAMES
---------   vaultwarden/server:latest   "/usr/bin/dumb-init …"   About a minute ago   Created   0.0.0.0:80->80/tcp, :::80->80/tcp, 3012/tcp   vaultwarden

**Comment by Masgalor** [comment]
Looks like you are trying to run docker inside LXC.
Could you elaborate more on your actual setup?

If my guess was right and you are running docker inside LXC, you should know that this has several limitations and problems.
And there is nothing vaultwarden can do about that.

**Reply by travisanderson65** [reply]
That explains it was was using an LXC. I will try with a virtual machine. Thanks
